Formula/cling.rb

```diff
@@ -0,0 +1,38 @@
+class Cling < Formula
+  desc "The cling C++ interpreter"
+  homepage "https://root.cern.ch/cling"
+  url "http://root.cern.ch/git/cling.git",
+      :revision => "bfe05be443c3f16b2eff4094384f1bd22480f3bc"
+  version "0.4"
```

Comment:
We'll need to get them to tag a version that works.

Reply:
They replied. A new version would be tagged in a few days.

Reply:
Hurray!

Reply:
@GauthamGoli was the new version tagged?

Reply:
No. I just pinged him on that thread.

Reply:
The maintainer's reply: https://root-forum.cern.ch/t/request-to-tag-a-version-of-cling-which-builds-on-macos/26225/8

Reply:
@GauthamGoli thanks for checking in with them!